dcpy/test/resources/mini_lion_prod_with_datasets/metadata.yml

```diff
@@ -0,0 +1,11 @@
+id: lion
+
+attributes:
+  display_name: LION Product Display Name
+  description: The lion dataset blah blah
+  publishing_frequency: "{{ lion_prod_level_pub_freq }}"
+  publishing_purpose: "legal compliance."
+
+datasets:
+  - pseudo_lots
+  - school_districts
```

Comment:
in product metadata, is `datasets` a list of all datasets or just those that should inherit attributes from the product?

as I type the question though, it does feels like all datasets would inherit attributes from the product lol

Reply:
yeah, that's a good question. It should be all datasets... which also implies that it's not enough to just have a folder with metadata. The Dataset has to be listed here, or it's like it doesn't exist to the product